helm-chart/binderhub/values.yaml

```diff
@@ -148,6 +148,13 @@ dind:
       tag: 17.11.0-ce-dind
   storageDriver: overlay2
   hostSocketDir: /var/run/dind
+  resources:
```

Comment:
can we expose these without setting defaults?

Reply:
e.g. an empty dict: `resources: {}`

Reply:
I think we can merge this once we set the defaults to empty.